.github/workflows/compare-ds.yaml

```diff
@@ -52,27 +52,21 @@ jobs:
           path: 'output.json'
           prop_path: 'product'
       - name: Build product ${{ github.base_ref }} (${{ steps.fork_point.outputs.FORK_POINT }})
-        if: ${{ steps.ctf.outputs.CTF_OUTPUT_SIZE != '0' }}
         run: ./build_product ${{steps.product.outputs.prop}} --datastream-only
```

Comment:
You will need to define a `product` to be built when there is no Content Test Filtering output.

Reply:
and whenever `${{steps.product.outputs.prop}}` is used.

Reply:
I just wanted to trigger `Compare DS` to see how the diff would look like.